new/LIT6843Commentary.xml

```diff
@@ -0,0 +1,97 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6843Commentary" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="en" xml:id="t1">Commentary on Old Testament books</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="AY"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>This work consists of a Gəʿəz to Gəʿəz commentary of  Old Testament books. It is attested in 
+                    <ref type="mss" corresp="MDA003"/>as a commentary of ten books, in <ref type="mss" corresp="EMML2101"/>as a commentary of five books.
+                    <ref type="mss" corresp="BNFabb156"> as a commentary of ten books</ref> </p>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="ChristianLiterature"/>
+                    <term key="Commentary"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="AY" when="2023-06-23">Created entity</change>
+            <change who="AY" when="2023-06-27">Discribe the conetnts  </change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography"><bibl>
+                <ptr target="bm:EMML6"></ptr>
+                <citedRange unit="page">196</citedRange>
+            </bibl></div>
+            <div type="edition">
```

Comment:
Two points: I would take as the reference manuscript EMML 2101 which is much older and closer to the original (?) than MDA003, and is already described by Getatchew Haile. He calles the collection "Geez commentary on the Pentateuch", that is, all books up up to "Numbers". All texts behind this one in MDA003 looks like additions. Also, looking at  "Maṣḥafa mənhar" I do not see that it is a commentary.  "Maṣḥafa miṭata bərhānāta samāy" is also not a commentary, probably they deserve separate treatment (IDs?). For the rest (Commentary to Maccabees and Daniel) please check if we already have them.  I remember I have recently created records for commentaries on prophetic books. You need to look up in A. Butts' recent articles on EMML 2101, and also in Cowley 1988 on biblical interpretation.